crates/db_schema_file/diesel_ltree.patch

```diff
@@ -17,3 +17,14 @@ index 255c6422..f2ccf5e2 100644
          creator_id -> Int4,
          post_id -> Int4,
          content -> Text,
+@@ -1333,8 +1333,10 @@ diesel::allow_tables_to_appear_in_same_query!(
+   registration_application,
+   report_combined,
+   search_combined,
+   site,
+   site_language,
+   tag,
++  person_actions,
++  image_details,
```

Comment:
diesel_cli was not automatically creating the lines for these two tables so I added them manually via patch. `person_actions` probably fails because it has two foreign keys to the same table (likely a bug in diesel), and image_details doesnt use any explicit foreign key for the join.